src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RankSumTest.java

```diff
@@ -67,12 +69,16 @@ public RankSumTest(){
             return Collections.emptyMap();
         }
 
+        final MannWhitneyU mannWhitneyU = new MannWhitneyU();
+
         // we are testing that set1 (the alt bases) have lower quality scores than set2 (the ref bases)
-        final double p = MannWhitneyU.runOneSidedTest(useDithering, altQuals, refQuals).getLeft();
-        if (Double.isNaN(p)) {
+        final MannWhitneyU.Result result = mannWhitneyU.test(Doubles.toArray(altQuals), Doubles.toArray(refQuals), MannWhitneyU.TestType.FIRST_DOMINATES);
+        final double zScore = result.getZ();
+
+        if (Double.isNaN(zScore)) {
             return Collections.emptyMap();
         } else {
-            return Collections.singletonMap(getKeyNames().get(0), String.format("%.3f", p));
+            return Collections.singletonMap(getKeyNames().get(0), String.format("%.3f", zScore));
```

Comment:
GATK4 definitely was not outputting p-values previously, since it was usually outputting negative values for the RankSum annotations :)